target/linux/ar71xx/image/generic.mk

```diff
@@ -722,6 +722,15 @@ define Device/xd3200
 endef
 TARGET_DEVICES += xd3200
 
+define Device/xd6700
```

Comment:
This definition is almost identical as above one (xd3200), you can limit duplicated lines and use the same approach as above (include xd3200 and overwrite different variables).

Reply:
Kernel output on booting:
[    0.000000] MyLoader: sysp=7a7b9d7d, boardp=6dfafbff, parts=fe6beeff
[    0.000000] bootconsole [early0] enabled
[    0.000000] CPU0 revision is: 00019374 (MIPS 24Kc)
[    0.000000] SoC: Qualcomm Atheros QCA9533 ver 2 rev 0
[    0.000000] Determined physical RAM map:
[    0.000000]  memory: 08000000 @ 00000000 (usable)
[    0.000000] Initrd not found or empty - disabling initrd